app/src/main/res/values/strings.xml

```diff
@@ -227,66 +227,66 @@
     <string name="telegram">Telegram</string>
     <string name="reddit">Reddit</string>
     <string name="twitter">Twitter</string>
-    <string name="turnInternetOn">Please connect to the internet by turning on WiFi or mobile data.</string>
-    <string name="open">Open …</string>
+    <string name="turnInternetOn">Please turn on Wi-Fi or mobile data to connect to the Internet.</string>
+    <string name="open">Open…</string>
     <string name="chapters">Chapters</string>
     <string name="change_playback_speed">Playback speed</string>
-    <string name="require_restart">Restart required</string>
-    <string name="require_restart_message">These changes might not be applied without restarting the app. Do you want to restart the app now?</string>
+    <string name="require_restart">App restart required</string>
+    <string name="require_restart_message">Restart the app ?</string>
     <string name="navLabelVisibility">Navbar label visibility</string>
     <string name="always">Always</string>
     <string name="selected">Selected</string>
     <string name="never">Never</string>
-    <string name="autoRotatePlayer">Auto fullscreen</string>
-    <string name="autoRotatePlayer_summary">Automatically switch to player fullscreen when the device gets turned.</string>
+    <string name="autoRotatePlayer">Auto-fullscreen</string>
+    <string name="autoRotatePlayer_summary">Fullscreen playback when the device gets turned.</string>
     <string name="pure_theme">Pure theme</string>
-    <string name="pure_theme_summary">Pure white/black theme</string>
+    <string name="pure_theme_summary">Only black and white</string>
```

Comment:
I suggest keeping the old term here too (`Pure white/black theme`)

Reply:
My thinking is that "Black and white" is a natural linguistic concept.
Moreover, it could be just "Black and white" in the title, without any summary.
This again begs the question of why it isn't a selection in the theme alternatives to begin with.
Because it isn't a black and white version of the selected theme(?)
Also both require an app restart, so no loss(?) making it one category.

On my device, the Appearance section scrolls by one entry.
Doing away with the toggle would help if not fix that completely.